app/models/ArrivalStatus.scala

```diff
@@ -57,6 +56,7 @@ object ArrivalStatus extends Enumerable.Implicits with MongoDateTimeFormats {
     }
   }
 
+  /*TODO: investigate this as part of a separate ticket*/
```

Comment:
Can we remove this when the ticket is created as this is live code

Reply:
Removing